CalibTracker/SiStripHitEfficiency/plugins/HitEff.cc

```diff
@@ -425,7 +426,26 @@ void HitEff::analyze(const edm::Event& e, const edm::EventSetup& es) {
             missHitPerLayer[14] += 1;
             hasMissingHits = true;
           }
+
+          //####### Consecutive missing hits case #######
+
+          //##### Layers from TIB + TOB
+          if (diffPreviousLayer == -3 && missedLayer > k_LayersStart && missedLayer < k_LayersAtTOBEnd &&
```

Comment:
Just checking: why not `<= k_LayersAtTOBEnd` ?

Reply:
That was a mistake, I will modify it !